src/plugins/navigation_embeddable/public/components/navigation_embeddable_panel_editor.tsx

```diff
@@ -112,48 +118,29 @@ const NavigationEmbeddablePanelEditor = ({
 
   const deleteLink = useCallback(
     (linkId: string) => {
+      if (orderedLinks.length <= 1) {
+        toasts.addDanger({
+          title: NavEmbeddableStrings.editor.panelEditor.getUnableToDeleteLinkToastTitle(),
+          text: NavEmbeddableStrings.editor.panelEditor.getMinimumLinksDeleteToastText(),
+        });
+        return;
+      }
```

Comment:
Is this necessary? Could we not just default back to the empty state? 🤔 cc @andreadelrio